net_test.go

```diff
@@ -0,0 +1,94 @@
+package openssl
```

Comment:
Please, don't test the private API, just a public API behavior.
```suggestion
package openssl_test
```

Reply:
Updated.